src/libraries/System.Text.Json/tests/Common/ReferenceHandlerTests/ReferenceHandlerTests.Deserialize.cs

```diff
@@ -22,15 +22,15 @@ private class EmployeeWithContacts
 
         #region Root Object
         [Fact]
-        public static void ObjectWithoutMetadata()
+        public async Task ObjectWithoutMetadata()
         {
             string json = "{}";
-            Employee employee = JsonSerializer.Deserialize<Employee>(json, s_deserializerOptionsPreserve);
+            Employee employee = await JsonSerializerWrapperForString.DeserializeWrapper<Employee>(json, s_deserializerOptionsPreserve);
```

Comment:
Nice. I was literally planning on making that change myself today. Great minds and all that